spec/u3d/installer_spec.rb

```diff
@@ -88,6 +88,25 @@ class DummyInstaller < U3d::BaseInstaller
           expect(installer.installed_sorted_by_versions).to eq(sorted_installed)
         end
       end
+
+      describe ".extra_installation_paths" do
+        it "converts paths to ruby paths" do
```

Comment:
I think it'd be better to have an intermediary `describe 'On Windows', if: WINDOWS do ... end` and `describe 'On other platforms', unless: WINDOWS do ... end` than handling the logic ourselves in the test. That's the convention we follow already for platform-dependent testing (see following test case).